lib/network.go

```diff
@@ -3809,6 +3856,30 @@ func (msg *MsgDeSoTxn) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
```

Comment:
Should this be in the mempool file?

Reply:
This should be deleted lol. It's unused. Better to do it in a separate PR tho.